docs/modelserving/storage/huggingface/hf.md

```diff
@@ -0,0 +1,113 @@
+# Deploy InferenceService with a saved model on Huggingface (HF)
+
+## Using Public HuggingFace Repo
+
+If no credential is provided, anonymous client will be used to download the model from HF repo.
+The uri is in the following format:
+
+
+```
+hf://${REPO}/${MODEL}:${HASH}(optional)
+```
+
+e.g. ```hf://facebook/opt-125m```
+
+## Using Private HuggingFace Repo
+
+KServe supports authenticating using HF_TOKEN
+
+## Option 1: Create HF Secret
```

Comment:
I think for this step is required regardless the options?

Option 1: Download the HF model using envFromSecret in kserve container
Option 2: Download the HF model using storage initializer